### ISSUE TITLE
Problem with latest kernal 5.0.0-25-generic

### ISSUE THREAD

**Issue by omalbastin** [lwfinger/rtlwifi_new#469]
Following is my kernal
`uname -r
5.0.0-25-generic
`
and i am using ubuntu 18.04.
I used to fix the rtl8723be driver issue in kermal 4.15. but after the update to 5.0.0, the fix doesnt affects now have only a weak signal

**Comment by lwfinger** [comment]
OK, you will need to redo all the ant_sel tests. Find where the option is stored using 'grep 8723 /etc/modprobe.d/*' and edit the file. If it is not there, you will need to create a file named /etc/modprobe.d/50-rtl8723be.conf containing the single line "options rtl8723be ant_sel=2".

The value of 2 is the most probable value for current kernels. Remember, for the option value to be registered, you will need to do a full shutdown after you change it. The device must be powered off!!!!!

**Comment by Kishore7298** [comment]
same problem here!
could you explain what to do in layman terms?

**Comment by omalbastin** [comment]
Now it worked. I cloned the latest master and installed it.

**Comment by lwfinger** [comment]
@Kishore7298 - I'm not sure what you mean by "layman" terms, but I will try. Your computer vendor, likely HP or Lenovo, saved a few cents by installing only one antenna in your device, even though there are two connectors. That would be OK if they had programmed the internal memory called EFUSE to indicate the connector that was used. They did not. This condition was likely found in testing under Windows and that driver was coded to ignore the indication from EFUSE. That is what needed to be done in the Linux driver. The initial attempt got it wrong. When that was fixed, the value for the ant_sel parameter changed.

@omalbastin - Are you aware that a simple 'git pull' command will get the latest code? It should not be necessary to reclone a repo unless you made the mistake of downloading a zip file!
